.gitignore

```diff
@@ -92,3 +92,6 @@ web/cmd/server/*.core
 
 # direnv (optional dev tool)
 .envrc
+
+# codegpt
+.codegpt/*
```

Comment:
I can remove this but I think it's a good addition so people don't commit their codegpt files